kolibri/core/assets/src/views/nav-bar/index.vue

```diff
@@ -343,16 +336,15 @@
       margin: 5px 0
       &:not(.is-divider)
         font-size: 14px
-        &.is-disabled
-          .ui-menu-option__icon
+        &.is-active
+          .ui-menu-option-icon
             color: $core-accent-color
           color: $core-accent-color
-          cursor: default
           font-weight: bold
           opacity: 1
-        .ui-menu-option__text
```

Comment:
Unfortunately moving the files into our source opens us up to our HTMLhint rules, which doesn't like underscores in class names.

The author of HTMLhint seems highly opposed to providing a directive/comment for ignoring files or lines:
 * https://github.com/yaniswang/HTMLHint/issues/23
 * https://github.com/yaniswang/HTMLHint/issues/109
 * https://github.com/yaniswang/HTMLHint/issues/138